src/shims/linux_stubs.h

```diff
@@ -16,6 +16,16 @@
 #ifndef __DISPATCH__STUBS__INTERNAL
 #define __DISPATCH__STUBS__INTERNAL
 
+#if !defined(TAILQ_FOREACH_SAFE)
```

Comment:
you haven't handled this review feedback.

Please use `#ifndef` and squash in a single commit before merge, thanks